pydoc/setup/MAC_INSTALL.md

```diff
@@ -0,0 +1,111 @@
+# MacOS
+
+Prerequisites for Arkouda can be installed using `Homebrew` or manually. Both installation methods have variances to account for the chipset being run.
+
+## Install Chapel
+
+It is recommended to compile Chapel from source. Alternatively, it can be installed via `Homebrew`.
+
+### Build from Source (Recommended)
+
+For convenience, the steps to install Chapel from source are detailed here. If you need more information, please visit the [Chapel Quickstart Guide](https://chapel-lang.org/docs/usingchapel/QUICKSTART.html).
+
+1) Download the current version of Chapel from [here](https://chapel-lang.org/download.html).
+
+2) Unpack the release
+
+```bash
+tar xzf chapel-1.29.0.tar.gz
+```
+
+3) Access the directory created when the release was unpacked
+
+```bash
+cd chapel-1.29.0
+```
+
+4) Configure environment variables. *Please Note: This command assumes the use of `bash` or `zsh`. Please refer to the [Chapel Documentation](https://chapel-lang.org/docs/usingchapel/QUICKSTART.html#quickstart-with-other-shells) if you are using another shell.*
+
+```bash
+source util/quickstart/setchplenv.bash
+```
+
+5) Update some environment variables to the recommended settings. *If you have not installed LLVM, set `CHPL_LLVM=bundled`. It is recommended to install LLVM using Homebrew, `brew install llvm`.* 
+
+6) Add the following to your `rc` file.
+
+```bash
+export CHPL_LLVM=system
+export CHPL_RE2=bundled
```

Comment:
we should note that `CHPL_GMP` needs to be set to whatever `CHPL_LLVM` is set to

Reply:
Updated